src/stats/inc/GaussianLikelihoodBlockDiagonalCovarianceRandomCoefficients.h

```diff
@@ -0,0 +1,83 @@
+//-----------------------------------------------------------------------bl-
+//--------------------------------------------------------------------------
+//
+// QUESO - a library to support the Quantification of Uncertainty
+// for Estimation, Simulation and Optimization
+//
+// Copyright (C) 2008-2015 The PECOS Development Team
+//
+// This library is free software; you can redistribute it and/or
+// modify it under the terms of the Version 2.1 GNU Lesser General
+// Public License as published by the Free Software Foundation.
+//
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+// Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public
+// License along with this library; if not, write to the Free Software
+// Foundation, Inc. 51 Franklin Street, Fifth Floor,
+// Boston, MA  02110-1301  USA
+//
+//-----------------------------------------------------------------------el-
+
+#ifndef UQ_GAUSSIAN_LIKELIHOOD_BLOCK_DIAG_COV_RAND_COEFFS_H
+#define UQ_GAUSSIAN_LIKELIHOOD_BLOCK_DIAG_COV_RAND_COEFFS_H
+
+#include <queso/GslBlockMatrix.h>
+#include <queso/GaussianLikelihood.h>
+
+namespace QUESO {
+
+/*!
+ * \file GaussianLikelihoodBlockDiagonalCovarianceRandomCoefficients.h
+ *
+ * \class GaussianLikelihoodBlockDiagonalCovarianceRandomCoefficients
+ * \brief A class representing a Gaussian likelihood with block-diagonal covariance matrix
```

Comment:
Ah, [I have one for `lnValue`](https://github.com/libqueso/queso/pull/322/files#diff-bc8a2baaedf5f6f99ccc57aafeaadc43R64), but I'll amend this brief for the sake of clarity (and also add a remark in the documentation for the constructor too).  Good catch.